examples/dialogs/dialogs/app.py

```diff
@@ -90,12 +90,53 @@ def action_save_file_dialog(self, widget):
         except ValueError:
             self.label.text = "Save file dialog was canceled"
 
+    def action_open_secondary_window(self, widget):
+        # Problems:
+        # - Setting secondary window's `app` manually
+        # - Secondary window's `icon` is not set
```

Comment:
I think this is a windows-only requirement; would it make sense to use the App's icon? Or do Windows apps have different icons for each window?

Reply:
I'm not sure what the recommendations on Form icons are. I would like to look at examples and see what common usage is, but I can't think of apps that have secondary windows. There are also different type of secondary windows available, including ones that are set to not have an icon:

> The Form class can be used to create standard, tool, borderless, and floating windows. You can also use the Form class to create modal windows such as a dialog box. 

I could just set the App's icon as each Form's icon for now.

Reply:
I added the app's icon to secondary windows in show().